core/connection.js

```diff
@@ -315,11 +314,11 @@ Blockly.Connection.prototype.canConnectWithReason_ = function(target) {
   if (this.isSuperior()) {
     var blockA = this.sourceBlock_;
     var blockB = target.getSourceBlock();
-    var superiorConn = this;
+    // var superiorConn = this;
```

Comment:
Why are these commented out. If this is intended, could you add a comment describing what you're doing here?

Reply:
This variable was used in a check further down that we disabled in pxt-blockly. I did not want to delete the variable, since we did not delete the check that was using it (we only commented it out).

I'll add a comment